### ISSUE TITLE
No action when bind onclick or bindPopup

### ISSUE THREAD

**Issue by GarryBrown** [bbecquet/Leaflet.PolylineDecorator#73]
I saw issue about  [FeatureGroup](https://github.com/bbecquet/Leaflet.PolylineDecorator/issues/44). It must work, but it's doesn't. Maybe i had a mistake.

**From dosc FeatureGroup**
```
L.featureGroup([marker1, marker2, polyline])
    .bindPopup('Hello world!')
    .on('click', function() { alert('Clicked on a member of the group!'); })
    .addTo(map);
```

**From your example with listener and bindPopup**
```
var arrow = L.polyline([[57, -19], [60, -12]], {}).addTo(map);
var arrowHead = L.polylineDecorator(arrow, {
      patterns: [
          { offset: '100%', repeat: 0, symbol: L.Symbol.arrowHead(
                   { pixelSize: 15, polygon: false, pathOptions: { stroke: true } }
             ) }
            ]
        })
            .bindPopup('Hello world!')
            .on('mouseover', function () { console.log('test') })
            .addTo(map);

```

**Comment by bbecquet** [comment]
Hi, and sorry for waiting so long.
You're right, it's using `LayerGroup` instances internally which don't respond to event binding. Using `FeatureGroup` everywhere solves the problem. I'll publish a fix soon.

**Comment by bbecquet** [comment]
It should be fixed in version 1.6.0. Thanks for your help! Don't hesitate to re-open if there are still bugs.
